ubuntu_18.04-dev.dockerfile

```diff
@@ -7,13 +7,18 @@ RUN apt-get install -y  \
     software-properties-common wget
 
 # pyne specific dependencies (excluding python libraries)
-RUN apt-get install -y build-essential git cmake vim emacs gfortran libblas-dev \
-                       python-pip liblapack-dev libeigen3-dev libhdf5-dev autoconf libtool
+RUN apt-get install -y build-essential git cmake gfortran libblas-dev \
+                       python3-pip liblapack-dev libeigen3-dev libhdf5-dev \
+                       hdf5-tools
```

Comment:
I would put all the command in line, something like:
```
build-essential \
git \
[...]
```
this would increase readability in the package that get installed